### ISSUE TITLE
Error when defining `@property`, Server Error -32000

### ISSUE THREAD

**Issue by synic** [proofit404/anaconda-mode#121]
This one is pretty easy to repro, it seems to happen to me any time I try to define a property.

I opened `~/foo.py` and started typing this:

```python
class Test(object):
    @property
    def hello(self):
```

**TYPE** this into a buffer where python-mode and anaconda-mode are active (don't paste it, it won't work).  As soon as you get to the first `(` after `hello`, things should start being extremely laggy.

In the `*Messages*` buffer, I see the following:

```
error in process filter: apply: Wrong type argument: stringp, ((data . "KeyError(u'property',)") (code . -32000) (message . "Server error"))
error in process filter: Wrong type argument: stringp, ((data . "KeyError(u'property',)") (code . -32000) (message . "Server error"))
error in process filter: apply: Wrong type argument: stringp, ((data . "KeyError(u'property',)") (code . -32000) (message . "Server error"))
error in process filter: Wrong type argument: stringp, ((data . "KeyError(u'property',)") (code . -32000) (message . "Server error"))
```

**Comment by proofit404** [comment]
Looks like `jedi` problem. Need dipper research.

**Comment by synic** [comment]
This seems to be a bigger issue than just the `@property` decorator.  I'm getting it on some of my other decorators:

```
error in process filter: apply: Wrong type argument: stringp, ((data . "KeyError(u'job',)") (code . -32000) (message . "Server error"))
error in process filter: Wrong type argument: stringp, ((data . "KeyError(u'job',)") (code . -32000) (message . "Server error"))
error in process filter: apply: Wrong type argument: stringp, ((data . "KeyError(u'job',)") (code . -32000) (message . "Server error"))
error in process filter: Wrong type argument: stringp, ((data . "KeyError(u'job',)") (code . -32000) (message . "Server error"))
```

**Comment by proofit404** [comment]
This errors doesn't change anything. You can try to reproduce this with `jedi.api` and see what happens directly.

**Comment by synic** [comment]
The errors don't break stuff, other than they make emacs incredibly laggy.  Whenever that happens, it's very hard to type, and every letter you type produces another error from anaconda-mode.

**Comment by proofit404** [comment]
I doesn't have much time to work on this issue. I'm sorry. No progress at all.

**Comment by dsedivec** [comment]
I think this might be the same problem I had, which I traced back to [Jedi issue #590](https://github.com/davidhalter/jedi/issues/590).  [Downgrading Jedi, as described in anaconda-mode's README](https://github.com/proofit404/anaconda-mode#attributeerror-and-keyerror-randomly-happens), fixed this problem for me.

Note that, if you're running a newer anaconda-mode, like HEAD, I believe those instructions may be out of date.  I've submitted PR #127 with updated directions.
